packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
 
-|Type     |Color (background/ icon & title)    |Icon    |Usage      |
-|---------|---------|--------|--------------|
-|Info     |Blue (#73bcf7/ #004368)    |fa-info-circle|Use for general informational messages (default)|
-|Warning  |Orange (#F0AB00/ #795600)  |fa-exclamation-triangle |Use to indicate that a non-critical error has occurred|
-|Critical |Red (#c9190b/ #a30000)     |fa-exclamation-circle | Use to indicate that a critical or blocking error has occurred
-|Success  |Green (#92D400/ #486b00)   |fa-check-circle | Use to indicate that a task or process has completed successfully
-|Default  |Cyan (#73C5C5/ #004368)   |fa-bell | Use for generic messages with no associated severity
 
 ## Usage
-There are two primary use cases when alerts are used:
+Alerts can be used to communicate:
 
-* **Synchronous notifications:** Synchronous notifications are returned as a direct response to some user action like clicking the Submit button on a form.
+* **Synchronous notifications:** Returned as a direct response to some user action like clicking the Submit button on a form.
 
-* **Asynchronous notifications:** Asynchronous notifications are generated by the system in response to some internal event. They may reflect status of a process, an internal error, or similar system event.
+* **Asynchronous notifications:** Generated by the system in response to some internal event. They may reflect status of a process, an internal error, or similar system event.
 
-Three alert patterns are defined in PatternFly for addressing these use cases.
+There are three ways that the system can communicate notifications to the users.
 
 |           |Usage     |UI placement     |Persistence
 |-----------|----------|-----------------|---------------|
-|[Inline alerts](#inline-alerts)|Synchronous notifications that are a direct response to a user action, eg., clicking the Submit button, and can be informational or can identify actions that are required on that page before the user can continue. |Appear at the top of the content area and push other content down. | Persist until the user dismisses them or navigates away from the page.
-|[Toast alerts](#toast-alerts) |A system notification that doesn’t interrupt the user’s current workflow. |Enter from the right edge of the screen and overlay content. |Remain until they time out, or the user dismisses them.
-|[Modal alerts](#modal-alerts) |Interrupting the flow until the user takes action. |Overlay the UI and prevent further user actions until the user closes the modal. |Yes, until the user dismisses the modal.
+|[Inline alerts](#inline-alerts)|Use for notifications that are a direct response to a user action, eg., clicking the Submit button, and can be informational or that identify actions required on that page before the user can continue. |Appear at the top of the content area and push other content down. | Persist until the user dismisses them or navigates away from the page.
```

Comment:
I would use "Example" instead of "eg" here (same comment as last documentation review!) and would also add a bit to that sentence to contextualize it "Example, clicking the submit button **in a wizard**"

Reply:
updated